_includes/item/metadata.html

```diff
@@ -16,9 +16,36 @@
             <a class="me-3" href="{{ t | strip | url_param_escape | prepend: '/browse.html#' | relative_url }}">{{ t | strip }}</a>
             {% endfor %}
             {% elsif f.external_link == "true" %}
-            <a href="{{ page[f.field] }}" target="_blank" rel="noopener">{{ page[f.field] }}</a>
+                {% assign links = page[f.field] | split: ";" %}
+                {% for markdown_link in links %}
+                {% if markdown_link contains "[" and markdown_link contains "]" and markdown_link contains "(" and markdown_link contains ")" %}
+                    {% assign link_text = markdown_link | split: '[' | last | split: ']' | first %}
+                    {% assign link_url = markdown_link | split: '(' | last | split: ')' | first %}
+                    {% if link_text == "" %}
+                    {% assign link_text = link_url %}
+                    <a class="me-3" href="{{ link_url }}" target="_blank" rel="noopener">{{ link_text }}</a>
+                    {% else %} 
+                    <a class="me-3" href="{{ link_url }}" target="_blank" rel="noopener">{{ link_text }} <svg class="bi icon-sprite" role="img" aria-label="external link">
+                        <use xlink:href="/assets/lib/cb-icons.svg#icon-external-link"/>
+                    </svg>
+                    </a>
+                    {% endif %}
+                {% else %}
+                    {{ markdown_link }}
+                {% endif %}
+                <br>
+                {% endfor %}
             {% else %}
-            {{ page[f.field] | replace: '""','"' }}{% endif %}
+            {% assign text = page[f.field] %}
+            {% assign urls = text | regex_match: '(https?:\/\/[^\s,]+)' %}
+            
+            {% for url in urls %}
```

Comment:
I think this should also be conditional on whether `{% elsif f.external_link == "true" %}`

Reply:
Here, however, the code does something different. With "external_links" it assumes a pure string or a tuple of label and uri. Here it simply replaces the all hyperlinks in a text with a-tags, e.g. 
`http://www.google.com`
becomes
`<a href="http://www.google.com" target='_blank' rel='noopener'>http://www.google.com</a>`
Maybe that is not necessary?

Reply:
It is necessary, but only if the option to turn url into clickable links is enabled. I haven't tested it. But it seemed off. 

You might want to move it inside the if clause.

Reply:
It works: I just added a test hyperlink in omeka on this mediaobject:

> Identifikator: m1234567 
> Titel: Tiff from sideload

Reply:
Ok, I'll try again. In my understanding external_links is an option offered by CB-CSV that turns strings with urls into clickable a-html-tags. If so, it should only do that if the option is true. So you'd need to add such a clause or integrate it above. Am I mistaken?

Reply:
> m1234567

Description should not include URLs but text only. See https://wiki.biozentrum.unibas.ch/display/dgpsgb/2024-05-03+Datenmodell+auf+Omeka+implementieren

Reply:
> > m1234567
> 
> Description should not include URLs but text only. See https://wiki.biozentrum.unibas.ch/display/dgpsgb/2024-05-03+Datenmodell+auf+Omeka+implementieren

also never ever general links that are not directly related to the resource, but for example link to the Handbuch or similar?

Reply:
That's what it says. All URL are to be added as a relation (Label/URI) and referenced as text reference on the label.

Reply:
I'm aware of that. but I don't quite understand the problem, because the code "doesn't hurt anyone". It simply detects hyperlinks in a text and doesn't encourage anyone to enter it wrong in omeka. 

It is also used for the field "Lizenz" to create a link. Otherwise we have to label the URI of the field licence in omeka as well and change it in config-metadata to "external_link", because these always have a label (see comment above).

Reply:
Here's the problem. Old behaviour: You turn off external links, urls are not clickable

<img width="731" alt="Screenshot 2024-05-24 at 14 23 38" src="https://github.com/Stadt-Geschichte-Basel/forschung.stadtgeschichtebasel.ch/assets/14755525/4469bd5d-c534-46f2-904c-2049cf292ab7">
<img width="1099" alt="Screenshot 2024-05-24 at 14 23 44" src="https://github.com/Stadt-Geschichte-Basel/forschung.stadtgeschichtebasel.ch/assets/14755525/b803353f-2907-4fb3-92da-eee858eba72c">

New behaviour: You turn off external links, urls are still clickable

<img width="980" alt="Screenshot 2024-05-24 at 14 23 59" src="https://github.com/Stadt-Geschichte-Basel/forschung.stadtgeschichtebasel.ch/assets/14755525/170fa575-7f8a-4675-b3ec-bbd7762f5598">
<img width="1206" alt="Screenshot 2024-05-24 at 14 24 07" src="https://github.com/Stadt-Geschichte-Basel/forschung.stadtgeschichtebasel.ch/assets/14755525/a5f0e241-4d46-46f7-8ca7-c542cc882f85">

Your PR breaks a feature. The user expects urls not to be clickable when set to external links is set to false.

Can you reproduce the bug?